pages/mesosphere/dcos/services/dse/3.1.0-6.7.6/updates/index.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: layout.pug
+navigationTitle: Updating 
+excerpt: Updating DSE
+title: Updating 
+menuWeight: 40
+model: /mesosphere/dcos/services/dse/data.yml
+render: mustache
+---
+
+<!-- #include /mesosphere/dcos/services/include/update.tmpl -->
+
+# Upgrading your cluster from {{ model.techShortName }} 5.1.10 to 6.7.6
+Due to the complexity of upgrading to {{ model.techShortName }} 6.7, we strongly advise that you attempt the upgrade on a test cluster before upgrading in your production environment. 
+
+Also we advise that you upgrade 5.1.10 directly to 6.7.6, skipping 6.7.2, as it has a problem in upgrading when TLS is enabled. 
+
+<p class="message--important"><strong>IMPORTANT: </strong> 
+<ul>
+<li>This action cannot be undone and you should plan for increased load activity on your cluster. This task should be scheduled for off-peak hours. Should any problems arise, <tt>pause</tt> the plan and investigate.</li>
+<li>Value of one of the properties <tt>dse_search</tt> or <tt>dse_analytics</tt> should be set to <tt>true</tt> before upgrading due to some limitation, later it can be set as needed. </li>
```

Comment:
> due to some limitation

what limitation?

Reply:
that's the issue when we are trying to upgrade sdk 42 to 57, these values changes are not acceptable.

Reply:
```suggestion
<li>You must set properties <tt>dse_search</tt> and <tt>dse_analytics</tt> to <tt>true</tt> before upgrading. Afterward, you can reset the configs to the desired value.</li>
```